Package.swift

```diff
@@ -0,0 +1,19 @@
+// swift-tools-version:4.0
```

Comment:
So what does this get us? A package just for testing? What's the end goal here?

Reply:
I’ve amended the PR to not expose _ProgressFraction but still build for Android.